Tasks/Gradle/task.json

```diff
@@ -210,7 +210,7 @@
             "type": "string",
             "label": "SonarQube Project Name",
             "required": true,
-            "helpMarkDown": "The SonarQube project name, i.e. sonar.projectName. If not set, the plugin will select its own default. See: http://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner+for+Gradle",
```

Comment:
Unrelated change - the plugin will not set anything by default because the project name, key and version are mandatory.